src/linter/RuleSet.zig

```diff
@@ -24,6 +27,27 @@ pub fn loadRulesFromConfig(self: *RuleSet, arena: Allocator, config: *const Rule
             });
         }
     }
+
+    {
+        var user_rule_iter = config._user_rules.map.iterator();
+        while (user_rule_iter.next()) |user_rule_entry| {
+            // FIXME: can we get a [:0]const u8 out of the json?
```

Comment:
Without allocating? Unlikely. These point to slices in a JSON string. Internal slices will not have a trailing sentinel.

Reply:
ngl I meant like some hacky shit where after you parse out the strings, you mutate the `"` character to a null char and cast it lmao